### DISCUSSION TITLE
Thanks given to me achievement badge

### DISCUSSION THREAD

**Discussion by James404-cyber** [community/community#31071] [Other Feature Feedback, Questions, & Ideas]
Thanks To The Hole Community.

**Comment by radar-blip** [comment]
The community says You're welcome! 😃

**Reply by AF250329** [reply]
You sure ? You sure that all `hole` community says: you are welcome ? or another `hole` community ? 😄 😄 😄

**Comment by MdHRShohel** [comment]
You're welcome from the community!

**Comment by James404-cyber** [comment]
❤

**Comment by tuves** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answering and locking the post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
